src/libs/EmojiUtils.js

```diff
@@ -319,7 +319,14 @@ function replaceEmojis(text, preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE,
     }
     for (let i = 0; i < emojiData.length; i++) {
         const name = emojiData[i].slice(1, -1);
-        const checkEmoji = trie.search(name);
+        let checkEmoji = trie.search(name);
+        if ((!checkEmoji || !checkEmoji.metaData.code) && lang !== CONST.LOCALES.DEFAULT) {
```

Comment:
I think we need a brief comment here explaining why we're doing this. I also have a slight preference of having `lang !== CONST.LOCALES.DEFAULT` first for readability (since we're only performing the rest if the locale is not default), but it's fine if you disagree!